docs/book/src/developer/providers/v1.0-to-v1.1.md

```diff
@@ -0,0 +1,21 @@
+# Cluster API v1.0 compared to v1.1
+
+## Minimum Go version
+
+- The Go version used by Cluster API is still Go 1.16+
+
+## Controller Runtime version
+
+- The Controller Runtime version is still v0.10.+
+
+## Controller Tools version (if used)
+
+- The Controller Tools version is still v0.7.+
+
+## Kind version
+
+- The KIND version used for this release is still v0.11.x
+
+## Deprecations and removals
+
+-
```

Comment:
could you kindly add a note for https://github.com/kubernetes-sigs/cluster-api/pull/5493 (not sure if there is something else that requires documentation)

Reply:
#5518 #5266 Aswell - they both deprecate a bunch of methods.

Reply:
Done. I'll also check other PRs. 
@killianmuldoon I'm not sure how #5533 is relevant.